packages/calcite-components/src/components/segmented-control/segmented-control.tsx

```diff
@@ -341,13 +341,13 @@ export class SegmentedControl
     }
   }
 
-  private handleDefaultSlotChange(event: Event): void {
+  private async handleDefaultSlotChange(event: Event): Promise<void> {
     const items = slotChangeGetAssignedElements(event).filter(
       (el): el is SegmentedControlItem["el"] => el.matches("calcite-segmented-control-item"),
     );
 
+    // await Promise.all(items.map((item) => item.componentOnReady()));
```

Comment:
remove comment :)

Reply:
Sorry about that. This is actually a fix I was testing to make sure the tests failed when they should.

Reply:
Actually, looks like I don't need it anymore after the latest fix. Will test again and remove if no longer needed.

Reply:
Scratch that. Just got a test failure after a few runs. Restored!